### ISSUE TITLE
Theme manager

### ISSUE THREAD

**Issue by roya0045** [qgis/QGIS#41597]
**Feature description.**
Working with many themes in a project can become cumbersome quickly, especially with the current control.

I think that having a dedicated theme manager that would show the associated layers only for each theme and add some control to add new layers, would go a long way.

This would help correcting themes without activating it and changing selected layers, and resaving.

**Comment by lieti** [comment]
I totally agree. Themes are nice in simple projects, especially when you are trying make map with inset.
But If you have big projects with many themes (or “views to data”) it is almost impossible to keep them actualized and "synchronized". If you are making little change of visualization in same layers it is necessary to make actualization of each theme. With current UI it is little bit complicated.
